.ci/pipelines/chaos_test_zeebe.groovy

```diff
@@ -0,0 +1,139 @@
+final PROJECT = "zeebe"
+final CHAOS_TEST_NAMESPACE = "zeebe-chaos-test"
+final NOTIFY_EMAIL = "maxim.danilov@camunda.com" // TODO change me
+
+pipeline {
+    triggers {
+        cron('H 1 * * *')
+    }
+
+    options {
+        buildDiscarder(
+                logRotator(
+                        numToKeepStr: '10',
+                        daysToKeepStr: '-1',
+                        artifactDaysToKeepStr: '-1',
+                        artifactNumToKeepStr: '10'))
+        timeout(time: 1, unit: 'HOURS')
+        skipDefaultCheckout()
+        timestamps()
+    }
+
+    agent {
+        kubernetes {
+            cloud "${PROJECT}-ci"
+            label "${PROJECT}-ci-build-${env.JOB_BASE_NAME}-${env.BUILD_ID}"
+            defaultContainer 'jnlp'
+            yaml pythonAgent(PROJECT)
+        }
+    }
+
+    stages {
+        stage('Clone') {
+            steps {
+                dir('zeebe-benchmark') {
+                    git url: 'git@github.com:zeebe-io/zeebe-benchmark',
+                            branch: "master",
+                            credentialsId: 'camunda-jenkins-github-ssh',
+                            poll: false
+                }
+                dir('zeebe') {
+                    git url: 'git@github.com:zeebe-io/zeebe',
+                            branch: "SRE-668", // TODO: change me
```

Comment:
I assume this should be develop then?

Reply:
Yes, this would be the last thing to change :) It's pointing to the current branch for the debugging purposes only.